src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -174,7 +174,7 @@ public long getIdLong()
     @Override
     public String getJumpUrl()
     {
-        return String.format("https://discordapp.com/channels/%s/%s/%s", getGuild() == null ? "@me" : getGuild().getId(), getChannel().getId(), getId());
+        return String.format("https://discordapp.com/channels/%s/%s/%s", isFromType(ChannelType.PRIVATE) ? "@me" : getGuild().getId(), getChannel().getId(), getId());
```

Comment:
Wouldn't this throw null pointer exception when type is `GROUP`? How about accessing guild if `getChannelType().isGuild()` is `true`?

Reply:
groups are not supported in v4 anymore and were probably even removed

Reply:
Yea, Groups were removed, just not the ChannelType enum constant.

Reply:
I might still go for isGuild() like suggested tho, considering theres also Store (which hopefully doesn't receive messages, but you never know)

Reply:
Definitely seems like `isFromGuild` is more appropriate/clear